databases/week1/HW.sql

```diff
@@ -0,0 +1,60 @@
+#1.Find out how many tasks are in the task table
+USE db_hyf;
+SELECT COUNT(*) FROM task;
+
+#2.Find out how many tasks in the task table do not have a valid due date
+USE db_hyf;
```

Comment:
You don't need to repeat `Use "database_name"` for every linr  and avoid committing empty or commented lines of code. Pls remove from your query